### DISCUSSION TITLE
Timestamp areas not fill automatically (created_at & updated_at)

### DISCUSSION THREAD

**Discussion by exsbandit** [coderflexx/laravel-ticket#26] [Q&A]
Hello, 
How can i fix ticket messages table timestamp areas, when i insert a new message to table its not take created_at time automatically.
I dont want to change anything in vendor folder.
Thanks for your helps.

**Comment by exsbandit** [comment]
> Hello, How can i fix ticket messages table timestamp areas, when i insert a new message to table its not take created_at time automatically. I dont want to change anything in vendor folder. Thanks for your helps.

I  already use like that;

`$inserted = DB::table('helpdesk_messages')->insertGetId(
                [
                    'user_id' => $user_id,
                    'helpdesk_ticket_id' => $helpdesk_ticket_id,
                    'message' => $this->message,
                    'created_at' => Carbon::now()
                ]
            );`

**Comment by ousid** [comment]
Hey, @exsbandit 

Can I ask, why are you using the `DB` Facade, instead of the model instance? Cause the `DB::insert` and `DB::insertGetId` can't run the observer who's responsible for the timestamp autofill the timestamps, or any related observers you are using.

I would suggest using the `Message` model, or if you are using your own custom methods/implementation. I advise you to extend the model, and make it your own.

```php
public function Message extends \Coderflex\LaravelTicket\Models\Message
{
  // ...
}
```
